ttnn/cpp/ttnn/operations/pool/avgpool/avg_pool.hpp

```diff
@@ -4,7 +4,25 @@
 
 #pragma once
 
-#include "tt_eager/tt_dnn/op_library/pool/average_pool.hpp"
+#include "tt_metal/host_api.hpp"
+#include "tensor/tensor.hpp"
+
+#include "tt_dnn/op_library/operation.hpp"
+
+namespace tt {
+namespace tt_metal {
+
+enum class PoolType {
+    AVG
+};
+
+Tensor avg_pool2d(const Tensor& input, const MemoryConfig& memory_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, const std::optional<DataType>& output_dtype = std::nullopt);
```

Comment:
why is this in tt::tt_metal namespace?

Reply:
The code was structured like that earlier, and I have not changed it.